README.md

```diff
@@ -1,2 +1,586 @@
+```
+virtctl console -ngreen vm-a
+Successfully connected to vm-a console. The escape sequence is ^]
+
+[fedora@vm-a ~]$ curl 203.203.0.4:9000/hostname
+webserver
+```
+
+Let's now provision a network policy to block this ! Please provision the
+[NetworkPolicy manifest](manifests/ns-isolation/05-network-policy-single-port.yaml)
+which **only** allows access to the port 9001 within the primary UDN of the
+`green` namespace; once you do, you'll see you can no longer access the
+webserver on the pod.
+
+```
+[fedora@vm-a ~]$ curl 203.203.0.4:9000/hostname
+
+```
+
+But, if you provision yet another
+[webserver pod](manifests/ns-isolation/06-new-webserver-pod.yaml)
+(just edit its name, and which port it listens to), and you'll see you can
+access it:
+```
+# example for a pod named new-webserver, which listens on port 9001
+kubectl get pods -ngreen new-webserver -ojsonpath="{@.metadata.annotations.k8s\.v1\.cni\.cncf\.io\/network-status}" | jq
+[
+  {
+    "name": "ovn-kubernetes",
+    "interface": "eth0",
+    "ips": [
+      "10.244.1.13"
+    ],
+    "mac": "0a:58:0a:f4:01:0d",
+    "dns": {}
+  },
+  {
+    "name": "ovn-kubernetes",
+    "interface": "ovn-udn1",
+    "ips": [
+      "203.203.0.8"
+    ],
+    "mac": "0a:58:cb:cb:00:08",
+    "default": true,
+    "dns": {}
+  }
+]
+
+virtctl console -ngreen vm-a
+Successfully connected to vm-a console. The escape sequence is ^]
+
+[fedora@vm-a ~]$ curl 203.203.0.8:9001/hostname
+new-webserver
+```
+
+## Cluster wide network
+In the diagram below, you will find the graphical depiction of the use case we
+are trying to achieve:
+![Cluster-wide Network](assets/cluster-wide-network.png)
+
+In it, we can see two cluster-wide networks - `happy`, and `sad` network.
+Traffic will be limited within each network - i.e. the workloads deployed in the
+namespaces interconnected by the `happy` network by the `happy` network (`red`
+and `blue`) will be able to communicate. Likewise for the namespaces
+interconnected by the `sad` network: `green` and `yellow`.
+
+What is not possible, is for a workload in the `red` namespace to communicate
+with a workload on the `green` namespace, since they are connected to different
+networks. And all of this, without having to provision a single NetworkPolicy.
+
+For this demo, we will focus exclusively on one of the namespace, the `happy`
+one,and we'll show something different - but all the features we've seen so far
+are be covered for this API as well.
+
+We will create a single cluster wide network, connecting two namespaces - the
+network will be called `happy`, and it will interconnect the `red` and `blue`
+namespaces.
+
+For that, provisioning the following yamls:
+- [namespaces and cluster-wide UDN](manifests/cluster-wide-network/01-udn.yaml)
+- [workloads](manifests/cluster-wide-network/02-workloads.yaml)
+
+This will also create 2 VMs, one in each namespace.
+
+We will now see they are both connected:
+```
+# let's get the VMs IPs
+kubectl get vmi -A -owide
```

Comment:
[02-workloads.yaml](https://github.com/maiqueb/fosdem2025-p-udn/blob/add-readme/manifests/cluster-wide-network/02-workloads.yaml) shows 4 running VMs, but here we see only the happy ones :-(.

Reply:
I'll tackle it in a follow-up ! 

Thanks for raising !